downloads/github-git-advanced-cheat-sheet.md

```diff
@@ -0,0 +1,45 @@
+# Advanced Cheat Sheet
+
+
+## Merge commands
+
+Abort the merge
+
+```$ git merge --abort```
+
+## Stash commands
+
+Save a stash with a particular name
+
+```$ git stash save <stash name>```
+
+Delete a particular stash
+
+```$ git stash pop <stash name>```
```

Comment:
Oops! I missed that :disappointed:  
I wanted to add the drop command. Thanks! 
I will fix it now and send a PR again.